Content.Server/Sound/EmitSoundSystem.cs

```diff
@@ -58,6 +59,8 @@ public override void Initialize()
             SubscribeLocalEvent<EmitSoundOnActivateComponent, ActivateInWorldEvent>(HandleEmitSoundOnActivateInWorld);
             SubscribeLocalEvent<EmitSoundOnTriggerComponent, TriggerEvent>(HandleEmitSoundOnTrigger);
             SubscribeLocalEvent<EmitSoundOnUIOpenComponent, AfterActivatableUIOpenEvent>(HandleEmitSoundOnUIOpen);
+            SubscribeLocalEvent<EmitSoundOnPickupComponent, GettingPickedUpAttemptEvent>(HandleEmitSoundOnPickup);
```

Comment:
This isn't the correct event to use, this is raised on actionblocker to check if pickup is possible and can be called at any time.

Reply:
Made a new one, on the same principle as drop